src/libs/Url.ts

```diff
@@ -29,12 +26,9 @@ function getPathFromURL(url) {
 
 /**
  * Determine if two urls have the same origin
- * @param {String} url1
- * @param {String} url2
- * @returns {Boolean}
  */
-function hasSameExpensifyOrigin(url1, url2) {
-    const removeW3 = (host) => host.replace(/^www\./i, '');
+function hasSameExpensifyOrigin(url1: string, url2: string): boolean {
+    const removeW3 = (host: string): string => host.replace(/^www\./i, '');
```

Comment:
```suggestion
    const removeW3 = (host: string) => host.replace(/^www\./i, '');
```

I think this function is to simple to specify the return type, wdyt @blazejkustra ?